lib/protocol/protocol.js

```diff
@@ -352,7 +354,7 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
       if (isErrorType(err, errors.ProxyRequestError)) {
```

Comment:
maybe it might be handy to just use duck typing here and check if the err object has the corresponding method/property